README.md

```diff
@@ -236,11 +229,9 @@ The default value set is to 4 (units are in MB ), this is used to configure the
 In case of Large messages , it is recommended to use streaming than setting a very high value on this configuration.
 It is not recommended to set the value more than 4GB
 
-* **monitoring_enabled** (optional; default: `true`) - 
-Enable or Disable monitoring of Requests arrived and response sent back
-
-* **monitoring_svc_end_point** (optional;only applies if `monitoring_enabled` is set to true) - 
-Needs to be a vaild url where the request and response stats are published as part of monitoring
+* **metering_enabled** (optional,default: `false`) -
+This is used to define if metering needs to be enabled or not .You will need to define a valid ` metering_end_point` 
```

Comment:
Am I right that `metering_end_point` description should be added to README.md as well?

Reply:
Done now, my bad